pkg/goprobe/writeout/handler.go

```diff
@@ -2,196 +2,97 @@ package writeout
 
 import (
 	"context"
-	"fmt"
 	"io/fs"
 	"time"
 
-	"github.com/els0r/goProbe/cmd/goProbe/config"
-	"github.com/els0r/goProbe/pkg/capture"
+	"github.com/els0r/goProbe/pkg/capture/capturetypes"
 	"github.com/els0r/goProbe/pkg/goDB"
 	"github.com/els0r/goProbe/pkg/goDB/encoder/encoders"
 	"github.com/els0r/goProbe/pkg/logging"
+	"golang.org/x/exp/slog"
 )
 
-const WriteoutsChanDepth = 100
-
-// writerCleanupCutoff governs when old DB writers are deleted from the map
-// of available writers
-const writerCleanupCutoff = 3
-
-type Handler struct {
-	// LastRotation is the time the handler last rotated all flow maps
-	LastRotation time.Time
+type Handler interface {
+	HandleWriteout(ctx context.Context, timestamp time.Time, writeoutChan <-chan capturetypes.TaggedAggFlowMap) <-chan struct{}
+}
 
-	writeoutsChan  chan *writeout
-	captureManager *capture.Manager
+const WriteoutsChanDepth = 100
 
+type GoDBHandler struct {
```

Comment:
I suggest that we put this into `godb.go` and leave the `handler.go` file lean as in: only defining the interface.

Reply:
Absolutely, done (and also added some comments and removed dead code).